File: 448-Find-All-Numbers-Disappeared-in-an-Array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        n = len(nums)
        nums = set(nums)
        res = set()

        for i in range(1, n + 1):
            if i not in nums:
                res.add(i)

        return res

File: neetcode/448-Find-All-Numbers-Disappeared-in-an-Array.ts
function findDisappearedNumbers(nums: number[]): number[] {
  const set: Set<number> = new Set(nums)
  const res: number[] = []

  for (let i = 1; i <= nums.length; i++) {
    if (!set.has(i)) res.push(i)
  }

  return res
};